Monika After Story/game/script-ch30.rpy

```diff
@@ -826,6 +826,585 @@ init python:
             setattr(store, word, value)
 
 
+init 995 python in mas_reset:
+    # define the decorator for reset code
+
+    import store.mas_submod_utils as mas_submod_utils
+
+    def ch30_reset(priority=9999):
```

Comment:
Maybe we could use negative priority for our resets? So people don't have to work with 5 digits.

Reply:
function plugins run before the code in a label, so submod's reset code could have expected our reset code to run after. the default priority of function plugins is 0, so by picking priority numbers larger than 0, existing submods using default priority wont have that assumption broken.

Reply:
This is a very niche case
- means people used fp for this label
- expected it to run first
- didn't set any priority, if they wanted to make sure it's being executed as early as possible, they would probably have set it? At least so it doesn't interfere with other fp callables

Those are factors that are very unlikely to come together if at all. Yet they are going to define our code in future, because we would have to do `@ch30_reset(10000)` (for example I needed to register a few funcs), it's completely unclear where 10k came from.

Reply:
I agree, sticking within the -999 - 999 range is more inline with the style of the rest of our codebase (i.e. init levels), especially in a paradigm where using inits in the 4 digit range (and above) is discouraged. It makes the most sense to use more traditional values, imo

Reply:
I don't agree that this would be a niche case, but since we've advertised the next few updates can have breaking changes, this will be one of them.